device/mellanox/x86_64-nvidia_sn5600-r0/Mellanox-SN5600-C256S1/buffers_defaults_t0.j2

```diff
@@ -15,10 +16,10 @@
     limitations under the License.
 #}
 {% set default_cable = '40m' %}
-{% set ingress_lossless_pool_size =  '118249472' %}
-{% set ingress_lossless_pool_xoff =  '15728640' %}
+{% set ingress_lossless_pool_size =  '115245056' %}
+{% set ingress_lossless_pool_xoff =  '14826496' %}
 {% set egress_lossless_pool_size =  '158229504' %}
-{% set egress_lossy_pool_size =  '118249472' %}
+{% set egress_lossy_pool_size =  '115245056' %}
 
 {% import 'buffers_defaults_objects.j2' as defs with context %}
```

Comment:
will this config create a single lossy buffer pool?

Reply:
yes, it will create a single egress & ingress lossy pool

Reply:
hi Noa, just to double confirm - will the lossless pool be removed?